cfme/common/__init__.py

```diff
@@ -305,14 +305,21 @@ class TaggableCommonBase(object):
     """Class represents common functionality for tagging via collection and entities pages"""
 
     def _set_random_tag(self, view):
-        random_cat = random.choice(view.form.tag_category.all_options).text
-        # '*' is added in UI almost to all categoly while tag selection,
+        if self.appliance.version > "5.11":
+            random_cat = random.choice(view.form.tag_category.all_options)
```

Comment:
optional: but just enthusiastic, is it not possible to implement `all_options` method of `ReactSelect` similar to `BootstrapSelect` means Options will be namedtuple of `text` and `web-element`?

Reply:
I agree with @digitronik here, is there a reason why this can't be done?

Reply:
Yeah this is a PITA, and obscure because its not really tied to the appliance version, its tied to which widget is being used